subiquity/ui/views/snaplist.py

```diff
@@ -102,48 +103,39 @@ def __init__(self, parent, snap, cur_channel):
                     is_classic=csi.confinement == "classic"))
             self.channels.append(Color.menu_button(Columns([
                 (channel_width, btn),
-                (max_version, Text(csi.version)),
-                (max_revision, Text("({})".format(csi.revision))),
-                (max_size, Text(humanize_size(csi.size))),
-                ('pack', Text(notes)),
+                (max_version,   Text(csi.version)),
+                (max_revision,  Text("({})".format(csi.revision))),
+                (max_size,      Text(humanize_size(csi.size))),
+                ('pack',        Text(notes)),
```

Comment:
This likely breaks under lint checks, right, the extra spaces after comma?

Reply:
"All checks have passed"